test/unit/app/sessionStoreTest.js

```diff
@@ -38,6 +38,7 @@ describe('sessionStore unit tests', function () {
     }
   }
   const fakeFiltering = {
+    clearHSTSData: () => {},
```

Comment:
it would be good to add a unit test to check that `clearHSTSData` is called if lastAppVersion is 0.21.x but not if it's 0.23.x